placement/storage/watch.go

```diff
@@ -34,35 +33,20 @@ var (
 
 type w struct {
```

Comment:
nit: You can just type alias it?

Reply:
I can't as I need to call the ValueWatch.Get() in the Get() function